### ISSUE TITLE
WARNING:tornado.access:404 GET when using panel.serve with prefix

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#1851]
I need to deploy some panel apps behind a reverse proxy and at a subpath `/my-panel-apps/`. I set the subpath using the `prefix` to  panel.serve.

#### Example

```python
import panel as pn

view = pn.pane.Markdown("Hello Panel World")

pn.serve({"": view}, port=80, prefix="my-panel-apps")
```

```bash
$ python 'scripts\prefix_serve.py'
Launching server at http://localhost:80
WARNING:tornado.access:404 GET /static/extensions/panel/css/card.css (::1) 0.50ms
WARNING:tornado.access:404 GET /static/extensions/panel/css/alerts.css (::1) 1.00ms
WARNING:tornado.access:404 GET /static/extensions/panel/css/json.css (::1) 1.00ms
WARNING:tornado.access:404 GET /static/extensions/panel/css/dataframe.css (::1) 0.50ms
WARNING:tornado.access:404 GET /static/extensions/panel/css/widgets.css (::1) 0.50ms
WARNING:tornado.access:404 GET /static/extensions/panel/css/markdown.css (::1) 276.50ms
```

**Comment by MarcSkovMadsen** [comment]
#### Additional Context.

Using `CDN` resources by setting `export BOKEH_RESOURCES=cdn` is not a working around as it does not work due to CORS.

![image](https://user-images.githubusercontent.com/42288570/102470963-e3e81380-4054-11eb-8f6d-c31e525285ff.png)

**Comment by philippjfr** [comment]
Fixed for `panel serve` in https://github.com/holoviz/panel/pull/1830, will need another fix for `pn.serve` though.

**Comment by philippjfr** [comment]
>Using CDN resources by setting export BOKEH_RESOURCES=cdn is not a working around as it does not work due to CORS.

Afaik this is a bug in Chrome and can be fixed by clearing out your cache.

**Comment by galewis2** [comment]
Great, thanks for this fix! Is it possible to get a tagged release so it's easier to deploy this? Thanks!

**Comment by philippjfr** [comment]
Looks like this was fixed by #1830 afterall.
